### ISSUE TITLE
"Link from" in issues: ability for source file selecting by its id

### ISSUE THREAD

**Issue by aviatorm** [danmunn/redmine_dmsf#1074]
Hello. I'm new dmsf user, and only starting with it. I want to use the feature "link from" in issues. And this worked by file title selecting only.
Now i see if large number of documents a located in some folder of dmsf, it is difficult find needed document by its title, because this fields may be often same or similar.

However, file id is unique. May be, ability for source file selecting by its id is available?

**Comment by picman** [comment]
Try to enable **Fast links** in _Project->Settings->DMS_.

**Comment by aviatorm** [comment]
Yes, i enabling this checkbox. Nothing is changed. Select window is same.

**Comment by picman** [comment]
You're right. The feature is available for **Link To** only.
